app/src/db/migrations/20210721210234_019-api-key-models.js

```diff
@@ -89,6 +89,7 @@ exports.up = function (knex) {
       );
       table.uuid('formId').references('id').inTable('form').notNullable().index();
       table.uuid('secret').unique().notNullable();
+      table.boolean('filesAPIAccess').defaultTo(false);
```

Comment:
For the database migrations, instead of changing the old migrations we create a new migration with `npm run migrate:make [your-migration-name]`. I don't think this change to an old migration would get picked up when deploying in dev/test/prod.

In the new migration we would do an add column for the "up" and then remove the column for the "down".

Reply:
Ok let me try it out.

Reply:
Hi Walter, I was having some issues with resolving the merge conflicts so I decided to create a new PR to keep it cleaner: https://github.com/bcgov/common-hosted-form-service/pull/1253